packages/core/test/CorePlugins.test.ts

```diff
@@ -0,0 +1,130 @@
+import { testEditor } from '../../utils/src/testUtils';
+import JWEditor from '../src/JWEditor';
+
+const indent = (editor: JWEditor) => editor.execCommand('indent');
+const outdent = (editor: JWEditor) => editor.execCommand('outdent');
+
+describe('core', () => {
+    describe('utils', () => {
+        describe('CorePlugin', () => {
+            describe('tab', () => {
```

Comment:
`describe('indent/outdent'` rather than 'describe('tab''?
Tab is just one way to indent/outdent. A button can also do the job.